ci-operator/config/openshift/cluster-api-provider-agent/openshift-cluster-api-provider-agent-master.yaml

```diff
@@ -129,6 +137,27 @@ tests:
       GATHER_CAPI_LOGS: "true"
       TEST_SUITE: full
     workflow: assisted-baremetal
+- as: e2e-assisted-operator-capi-periodic
+  cron: '@daily'
+  steps:
+    cluster_profile: packet-assisted
+    dependencies:
+      ASSISTED_OPENSHIFT_INSTALL_RELEASE_IMAGE: release:latest
+      HYPERSHIFT_IMAGE: hypershift-operator
+      INDEX_IMAGE: assisted-service-operator-index
+      OPENSHIFT_INSTALL_RELEASE_IMAGE: release:latest
+      PROVIDER_IMAGE: pipeline:cluster-api-provider-agent
+    env:
+      DEVSCRIPTS_CONFIG: |
+        EXTRA_WORKER_VCPU=8ci-operator/jobs/openshift/cluster-api-provider-agent/openshift-cluster-api-provider-agent-master-periodics.yaml
+        EXTRA_WORKER_MEMORY=32768
+        EXTRA_WORKER_DISK=120
+        PROVISIONING_NETWORK_PROFILE=Disabled
+        REDFISH_EMULATOR_IGNORE_BOOT_DEVICE=True
```

Comment:
I think those are the default values already. see:
https://github.com/openshift/release/blob/08e7671ec8e7ee3db6bbc04ed573a01996d186cb/ci-operator/step-registry/assisted/baremetal/operator/capi/assisted-baremetal-operator-capi-workflow.yaml#L14-L19